Docs/Building.md

```diff
@@ -56,11 +57,36 @@ cmake -DOPENSSL_ROOT_DIR="/usr/local/opt/openssl" -G "Xcode" ..  # macOS
 
 While most dependencies for macOS and Windows are included in the repo, there are some linux libraries which cannot be included in the repo.
 
+For Pop_OS!, install the `Debian` dependencies, then refer to the [Pop_OS! Notes](#Pop_OS-Notes) section below.
+
 - Debian: `apt install build-essential libssl-dev libx11-dev libxrandr-dev libcurl4-openssl-dev libglu1-mesa-dev libpulse-dev libogg-dev libasound-dev libjack-dev`
 - Fedora: `dnf install openssl-static libX11-devel libcurl-devel mesa-libGLU-devel libXrandr-devel libogg-devel pulseaudio-libs-devel alsa-lib-devel jack-audio-connection-kit-devel`
 - Arch: `pacman -S openssl libx11 libxrandr curl mesa glu libogg pulseaudio jack`
 - Alpine: `apk add build-base openssl-dev libx11-dev libxrandr-dev curl-dev mesa-dev glu-dev pulseaudio-dev libogg-dev alsa-lib-dev jack-dev`
 
+### Pop_OS! Notes
+
+More recent builds of Pop_OS! are missing some required dependencies.
+
+#### Python 2.7
+
+The build requires Python 2.7 be installed and symlinked as `python`.
+
+Run `python --version`. If it's ok, skip this section. If `python --version` says `command not found`, run the following:
+
+```
+sudo apt install -y python2.7
+sudo ln -s /usr/bin/python2.7 /usr/local/bin/python
+```
+
+#### clang
+
+`clang++` is needed for compiling the source code.
+
+```
+sudo apt install -y clang
+```
```

Comment:
Worth calling out here or should I just move it into the Debian dependencies above?

Reply:
clang appears to be a general linux dependency for crashpad related reasons

maybe

Reply:
@poco0317 I uninstalled `clang` and built without crashpad and didn't get any errors. I'll update the readme